enhancements/storage/storage-hypershift.md

```diff
@@ -0,0 +1,973 @@
+---
+title: storage-hypershift
+authors:
+- "@jsafrane"
+reviewers:
+- "@gnufied"
+- "@dobsonj"
+- "@csrwng"
+approvers:
+- "@derekwaynecarr"
+api-approvers:
+ - None
+creation-date: 2022-07-26
+last-updated: 2022-07-27
+tracking-link:
+- https://issues.redhat.com/browse/STOR-962
+see-also:
+replaces:
+superseded-by:
+---
+
+# Running storage operators in HyperShift management cluster
+
+## Summary
+
+This enhancement describes how OCP storage components will run in HyperShift's managed control plane and in the guest
+cluster.
+
+The components currently are:
+
+* cluster-csi-snapshot-controller-operator and its operand, csi-snapshot-controller.
```

Comment:
nit: can we follow always the same dash convention for naming? "cluster-csi-snapshot-controller-operator" vs "cluster-storage operator" vs "AWS EBS CSI driver"?

Reply:
AWS EBS CSI driver is actually collection of many things (its Deployment has 6 sidecars), it's not a monolith like one operators in one repo / Deployment.